security/keycloak-authz/pom.xml

```diff
@@ -13,7 +13,7 @@
     <dependencies>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-resteasy</artifactId>
+            <artifactId>quarkus-resteasy-reactive</artifactId>
```

Comment:
My only concern here is that we're losing the coverage of RestEasy Classic. 
Not sure if we should eventually start splitting the coverage into classic and reactive.

Reply:
This is concerning me too, let's discuss options on chat.